walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -56,6 +57,11 @@ public void setUp() {
     query = new FindMeetingQuery();
   }
 
+  @AfterClass
+  public static void tearDownClass() {
+    System.out.println("Executed after all");
+  }
```

Comment:
I think this may be left after debugging or experimenting. You don't need this.
In general tearDownClass is useful when testing something that has a dependency that requires some nontrivial amount of cleanup, which here is not the case. Examples: say you open a specific file to be used from all the unit tests which needs to be closed after all of them have ran, or some fake implementation of a service that takes tens of seconds to initialize and can be shared across unit tests, then it makes sense to initialize it once, re-use it and clean it up afterwards.

Reply:
Oops, I was experimenting with jUnit stuff for my presentation and forgot to delete these functions.